README.md

```diff
@@ -16,8 +16,16 @@ http://opensource.addepar.com/ember-charts/
 
 http://emberjs.jsbin.com/rekawobugu/1/edit
 
+Unfortunately, this version of Ember Charts is out of date,
+but the current maintainers of Ember Charts at Addepar have not been
```

Comment:
Well, I guess we can change "but" to "add". We're already saying that we have no idea how to update that page, which is true as I tried and failed to update it once. I guess there is no need to soft-pedal or apologize for our failure. :)

I think only Alex (@azirbel) would know how to update the page; he was the person who first  added the JSBin example and did most of the work keeping it up to date, per the git commit history. I probably didn't do a good job fixing all the dependencies and that's why I couldn't figure out how to successfully update the page myself.